components/base/emit-blur-event.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Emit a `blur` event.
+ * If you call this function directly from outside the scope of a component, then you must bind the scope manually.
+ * @param {*} eventData The event data from the native DOM `blur` event
+ */
+export function emitBlurEvent (eventData) {
+  this.$emit('blur', eventData)
+}
+
+/**
+ * Adds the `emitBlurEvent` method
+ */
+export const EmitBlurEventMixin = {
+  methods: {
+    emitBlurEvent

```

Comment:
not a big deal, but why split implementation from mixin  ?

Reply:
I thought that it might be possible that we would want to access the function directly for some reason. JS benefits from having features of a functional language so I usually try to make functions directly available.

Reply:
ok